src/utils.ts

```diff
@@ -9,6 +9,7 @@ const ENC_BLOCK_SIZE = 24
 const PAD_FIRST_BYTE = 128
 const B64 = 'base64pad'
 const B58 = 'base58btc'
+const B64_URL = 'base64urlpad'
```

Comment:
base64url used in JOSE doesn't use padding.
```suggestion
const B64_URL = 'base64url'
```